samples/nfc/tag_reader/src/main.c

```diff
@@ -0,0 +1,197 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <drivers/st25r3911b_nfca.h>
+
+#define NFC_T2T_READ_CMD 0x30
+#define NFC_BLOCK_TO_READ_NUM 0
+
+#define NFC_T2T_READ_CMD_LEN 0x02
+#define NFC_T2T_BLOCK_LEN 0x10
+
+#define NFC_TX_DATA_LEN NFC_T2T_READ_CMD_LEN
+#define NFC_RX_DATA_LEN NFC_T2T_BLOCK_LEN
+
+#define TRANSMIT_DELAY 3000
+#define ALL_REQ_DELAY 2000
+#define T2T_READ_CMD_FDT 2068
+
+static u8_t tx_data[NFC_TX_DATA_LEN] = {NFC_T2T_READ_CMD, NFC_BLOCK_TO_READ_NUM};
+static u8_t rx_data[NFC_RX_DATA_LEN];
+
+static struct k_poll_event events[NFCA_EVENT_CNT];
+static struct k_delayed_work transmit_work;
+
+struct nfca_buf tx_buf = {
```

Comment:
`static` missing (and perhaps `const`, see the comment regarding parameters of `st25r3911b_nfca_transfer`). Same story with `rx_buf`.

Reply:
Fixed